portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -25,25 +27,28 @@
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
```

Comment:
Consider moving this new code to a separate Servlet or changing the name here, it's a bit misleading now :)